history/history-impl/src/main/java/com/duckduckgo/history/impl/RealNavigationHistory.kt

```diff
@@ -57,4 +62,16 @@ class RealNavigationHistory @Inject constructor(
     override suspend fun clearOldEntries() {
         historyRepository.clearEntriesOlderThan(currentTimeProvider.localDateTimeNow().minusDays(30))
     }
+
+    override fun isHistoryUserEnabled(): Boolean {
```

Comment:
I don't think so, any feature dealing with history would need access to these methods, so the cleanest the API we give them, the better, IMO. Also, I think that would be a bit confusing, since, as I mentioned on my previous comment, we usually reserve the `*Feature` suffix for RC-related stuff. Also note that this method is related to user settings, not RC